test/e2e/main_test.go

```diff
@@ -253,6 +253,27 @@ func TestManualMeteringInstall(t *testing.T) {
 			},
 			MeteringConfigManifestFilename: "node-selector-prometheus-importer-disabled.yaml",
 		},
+		{
+			Name:                      "ValidHDFS-MySQLDatabase",
+			MeteringOperatorImageRepo: meteringOperatorImageRepo,
+			MeteringOperatorImageTag:  meteringOperatorImageTag,
+			Skip:                      false,
```

Comment:
Probably should change this to `!runAllInstallTests` in the future once we add some periodic testing support, but this is fine for now.